CMakeLists.txt

```diff
@@ -159,12 +159,12 @@ endif()
 
```

Comment:
We should probably rename `NovelRTLib` to `libNovelRT` as that matches the "normal" naming convention